docs/statistics.md

```diff
@@ -0,0 +1,561 @@
+
+The following API functions can be used to retrieve statistics on an SRT socket.
+Refer to the documentation of the [API functions](API-functions.md) for usage instructions.
+* `int srt_bstats(SRTSOCKET u, SRT_TRACEBSTATS * perf, int clear)`
+* `int srt_bistats(SRTSOCKET u, SRT_TRACEBSTATS * perf, int clear, int instantaneous)`
+
+
+# Total accumulated measurements
+
+## msTimeStamp
+
+Time elapsed since the SRT socket was started (after successful call to `srt_connect(...)` or `srt_bind(...)`), in milliseconds.
+
+## pktSentTotal
+
+The total number of sent data packets, including retransmitted packets. Applicable for data sender.
+
+## pktRecvTotal
+
+The total number of received packets, including received retransmitted packets. Applicable for data receiver.
+
+## pktSndLossTotal
+
+The total number of data packets considered or reported lost (sender side). Does not correspond to packets detected as lost at receiving side. Applicable for data sender.
+
+A packet is considered lost in two cases: 
+1. Sender receives a loss report from receiver.
+2. Sender initiates retransmission after not receiving ACK for a certain timeout. Refer to `FASTREXMIT` and `LATEREXMIT` algorithms.
+
+## pktRcvLossTotal
+
+The total number of data packets detected lost on the receiver's side.
+
+Includes packets that failed to be decrypted (only as of SRT version 1.4.0).
+
+If a packet was received out of order, the gap (sequence discontinuity) is also 
+treated as lost packets, independent of the reorder tolerance value.
+
+Loss detection is based on gaps in Sequence Numbers of SRT DATA packets. Detection 
+of a packet loss is triggered by a newly received packet. An offset is calculated 
+between sequence numbers of the newly arrived DATA packet and previously received 
+DATA packet (the received packet with highest sequence number). Receiving older 
+packets does not affect this value. The packets from that gap are considered lost, 
+and that number is added to this `pktRcvLossTotal` measurement. In the case where 
+the offset is negative, the packet is considered late, meaning that it was either 
+already acknowledged or dropped by TSBPD as too late to be delivered. Such late 
+packets are ignored.
+
+## pktRetransTotal
+
+The total number of retransmitted packets. Calculated on the sender's side only. 
+Not exchanged with the receiver.
+
+## pktSentACKTotal
+
+The total number of sent ACK packets. Applicable for data sender.
+
+## pktRecvACKTotal
+
+The total number of received ACK packets. Applicable for data receiver.
+
+## pktSentNAKTotal
+
+The total number of NAK (Not Acknowledged) packets sent. Essentially means LOSS 
+reports. Applicable for data sender.
+
+## pktRecvNAKTotal
+
+The total number of NAK (Not Acknowledged) packets received. Essentially means 
+LOSS reports. Applicable for data receiver.
+
+## usSndDurationTotal
+
+The total accumulated time in microseconds, during which the SRT sender has some 
```

Comment:
Just to be sure, is it "microseconds" or "milliseconds"?

Reply:
It is in microseconds.
The "us" prefix is an ASCII version of "μs" for microseconds.